docs/community/contribute.md

```diff
@@ -27,45 +27,157 @@ Quick reference:
 git clone https://github.com/<your-username>/ragna.git
 ```
 
-### Set up development environment
+### Set up development environment and run Ragna
 
-We recommend using conda, but you can use any package and environment manager you
-prefer. The `environment-dev.yml` file at the root of the project lists all the required
-development dependencies.
+We use [Pixi](https://pixi.sh/dev/) to manage development environments with Ragna. The
+required development dependencies are listed in the `pyproject.toml` file at the root
+directory of the project.
 
-Create and activate a conda environment:
+Ragna has three main development environments: `dev-all-py310`, `dev-all-py311`, and
+`dev-all-py312`, which use Python 3.10, 3.11, and 3.12, respectively. The environment
+`dev-all` is the same as `dev-all-py310`. The environment `dev-all-py311` is used as an
+example in the sections below, but you can use any of the three Python versions you
+like.
+
+To install and activate a development environment, run
 
 ```bash
-conda env create --file environment-dev.yml
-conda activate ragna-dev
+pixi shell -e dev-all-py311
```

Comment:
Can we use 
```suggestion
pixi shell -e dev-all
```

for the examples, please? What is the point of this shortcut if we are not using it.